src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -55,9 +70,47 @@ export function libraryCurrent() {
   });
 }
 
+export function libraryPagerGen() {
```

Comment:
The alternative is to have multiple pager buttons in the liquid template that are hidden / revealed / changed / activated / deactivated depending on filter or pager state. This is much more concise, @cvalarida - will add notes to code.

Reply:
Yeah, I wasn't sure which would be cleaner. I don't like it, but it looks mostly harmless. :+1:

(Both _Hitchhiker's Guide to the Galaxy_ references intended. :wink:) 

Ideally, this would all be React, I think, but I understand there are other hurdles there. :man_shrugging: